common/argo/source/base/argo-server/argo-server-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argo-server
+spec:
+  selector:
+    matchLabels:
+      app: argo-server
+  template:
+    metadata:
+      labels:
+        app: argo-server
+    spec:
+      serviceAccountName: argo-server
+      containers:
+        - name: argo-server
+          image: argoproj/argocli:v2.12.3
```

Comment:
Should we try 2.12.5 now that it's out?

Reply:
@rushtehrani Absolutely, give me a few minutes to get that in.

Reply:
@rushtehrani Updated. Doing some testing.